### DISCUSSION TITLE
PaddleOCR/main/FAQ

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14006] [Q&A]
# PaddleOCR/main/FAQ

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/main/FAQ.html

**Comment by danshuiyu66617** [comment]
您好！PaddleOCR在ARM架构的CPU下性能如何？与X86架构同等规格CPU性能有多大的差距。

**Reply by Liyulingyue** [reply]
可以运行，暂时没有详细的对比数据。

**Comment by HsullGe** [comment]
代码格式以及数据都没有进行过改动 请问出现这个报错 应该如何去修改代码 AttributeError: 'list' object has no attribute 'reshape'

**Comment by xuelainiao** [comment]
您好！
希望GPU模式可以支持50系显卡，50系显卡使用用户已经非常多了，几经尝试都无法运行成功，希望可以得到支持，谢谢！
跪求跪求

**Comment by sijidoubi** [comment]
你好请问一下，PP-structureV3产线里面识别印章把印章的环形文字矫正用于后续文字识别。用的是TPS还是基于模型的方法呢？
